packages/datagateway-dataview/src/page/pageSearch.component.tsx

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import { TextField, InputAdornment } from '@material-ui/core';
+import SearchIcon from '@material-ui/icons/Search';
+import { StateType } from 'datagateway-common/lib/state/app.types';
+import { ThunkDispatch } from 'redux-thunk';
+import { pushPageSearch } from 'datagateway-common';
+import { AnyAction } from 'redux';
+import { connect } from 'react-redux';
+
+import debounce from 'lodash.debounce';
+
+interface PageSearchDispatchProps {
+  pushSearch: (search: string | null) => Promise<void>;
+}
+
+interface PageSearchStateProps {
+  search: string | null;
+}
+
+type PageSearchCombinedProps = PageSearchDispatchProps & PageSearchStateProps;
+
+// TODO: This will need to use data-search components from
```

Comment:
We don't want to remove the file, since the reason it's removed from dataview is more that we can't get this working yet, not that we'll never want the component.